test/utilities/connectivity_testing.go

```diff
@@ -237,15 +239,20 @@ func CreateGatewayEp(networkId string, ipAddress string, ipv6Adress string) erro
 	if err != nil {
 		return fmt.Errorf("Gateway Endpoint Not Found: %v", err)
 	}
-	ep.HostAttach(1)
+	if err := ep.HostAttach(1); err != nil {
+		return fmt.Errorf("Failed to attach HNS endpoint %q to host: %s", createdEp.Name, err)
+	}
 
 	//Hard Code for now
 	vNicName := fmt.Sprintf(`"vEthernet (%s)"`, gwEp.Name)
 	vEthernet := `"vEthernet (Ethernet)"`
 	os.Setenv("vEthernet", vEthernet)
 	os.Setenv("vNicName", vNicName)
-	cmd := exec.Command("cmd", "/c", "netsh", "int", "ipv4", "set", "int", "%vNicName%", "for=en")
-	cmd.Run()
+	command := []string{"cmd", "/c", "netsh", "int", "ipv4", "set", "int", "%vNicName%", "for=en"}
+	cmd := exec.Command(command[0], command[1:]...)
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("Failed to run command %s: %s", strings.Join(command, " "), err)
+	}
 	if err != nil {
```

Comment:
This error check is not right. You already have an error check at line:239

Reply:
Good catch, the reason the linter was complaining was that the original code was [not assigning the error from `Cmd.Run()`](https://github.com/microsoft/windows-container-networking/blob/master/test/utilities/connectivity_testing.go#L248) (should be `err = cmd.Run()`), which is why the linter complained and I initially just blindly added an extra error check, my bad!